codec-http/src/main/java/io/netty/handler/codec/rtsp/RtspResponseStatuses.java

```diff
@@ -23,270 +23,370 @@
 public final class RtspResponseStatuses {
 
```

Comment:
Please revert changes in this class as these are not needed for the functionality